order.json

```diff
@@ -4,7 +4,8 @@
   "contributing",
   {
     "about-jquery": [
-      "how-jquery-works",
+      "how-to-setup-jquery",
```

Comment:
We will want to handle a redirect for this I imagine.  That should block this being deployed to live.  I can handle landing together when this PR is ready.